docs/emr/presto-admin-installation-on-emr.rst

```diff
@@ -0,0 +1,32 @@
+=========================
+Presto Admin Installation 
```

Comment:
If it's just that one thing, I think leave it out.  there's already a page about online vs. offline installer.  Maybe just add a link to the regular installation page that links to the advanced installation options about online vs. offline installer.  It's not a problem that's llmited to emr